package-lock.json

```diff
@@ -32442,6 +32442,15 @@
         "node": ">=18.8.0"
       }
     },
+    "packages/plugin-types-bundler/node_modules/@types/node": {
```

Comment:
https://github.com/grafana/plugin-tools/actions/runs/14857769673/job/41715044403?pr=1765#step:4:10 

was forced to do this, even though the changes are not touching anything npm related...